File: source/com/barrybecker4/puzzle/common/solver/AStarConcurrentPuzzleSolver.java
package com.barrybecker4.puzzle.common.solver;

import com.barrybecker4.common.search.SearchSpace;

/**
 * @author Barry Becker
 */
public class AStarConcurrentPuzzleSolver<P, M> extends AStarPuzzleSolver<P, M> {

    public AStarConcurrentPuzzleSolver(SearchSpace<P, M> searchSpace) {
        super(searchSpace);
    }
}

File: scala-source/com/barrybecker4/puzzle/common/solver/AStarConcurrentPuzzleSolver.scala
package com.barrybecker4.puzzle.common.solver

import com.barrybecker4.search.space.SearchSpace


/**
  * @author Barry Becker
  */
class AStarConcurrentPuzzleSolver[P, M](searchSpace: SearchSpace[P, M])
  extends AStarPuzzleSolver[P, M](searchSpace) {
}
